server/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "utseus_api",
```

Comment:
using utseus in the name is a terrible choice. generally package should match the name of the repo. It would make more sense to have:

```
skills-server
skills-ui
```

and package.json:name to follow this naming.

Reply:
Done. 

I changed names inside `package.json` files. Do you want to say `package.json` also should be renamed?